src/sas/qtgui/Plotting/PlotUtilities.py

```diff
@@ -42,6 +42,25 @@
         ('Custom', 'x'),
 ])
 
+COLORS_LETTER = OrderedDict([
+        ('b', '#1f77b4'),
+        ('g', '#2ca02c'),
+        ('r', '#d62728'),
+        ('c', '#17becf'),
+        ('m', '#e377c2'),
+        ('y', '#bcbd22'),
+        ('k', '#000000'),
+])
+
+
+# MPL font weights list
+WEIGHTS = ['light', 'normal', 'regular', 'book', 'medium', 'roman',
```

Comment:
Do we need all these? Are they even supported on most systems?

Reply:
modified so the list is same as in 4.x